### ISSUE TITLE
Tag filter is not visible in the search results

### ISSUE THREAD

**Issue by Rob--W** [mozilla/addons-frontend#10789]
The list of search results can be filtered by tag, but this is not apparent from the UI.

For example, view the list of add-ons tagged `privacy` by visiting:
https://addons.allizom.org/en-US/firefox/search/?tag=privacy
Search for something else, e.g. "privacy". The URL becomes
https://addons.allizom.org/en-US/firefox/search/?q=privacy&tag=privacy

Both pages are similar, except for the list of results. The visibility of the tag filter should be improved, e.g. by changing `N results found for "(query)"` to `N results found for "(query)" filtered by "(tag, optionally with link to full list of add-ons by tag)", and/or by showing the tag filter(s) in the "Filter results" section at the left.

**Comment by Rob--W** [comment]
@eviljeff I filed this bug as a follow-up to your demo at the Add-ons Show and Tell.

I cannot edit the project on this issue, but it should be associated with https://github.com/orgs/mozilla/projects/158

**Comment by willdurand** [comment]
I don't think we expose all the possible filters on purpose, and we probably don't need to update the page title if the filter can be used without manually editing the URL. cc @jvillalobos

**Comment by willdurand** [comment]
I don't think we expose all the possible filters on purpose, and we should probably not update the page title if the filter cannot be used without manually editing the URL. cc @jvillalobos

**Comment by eviljeff** [comment]
we're going to be linking to the search pages with `?tag=xxx` in them soon, fwiw.

**Comment by jvillalobos** [comment]
We should probably do that same thing we do for categories, where using the search box removes the category filter.

**Comment by eviljeff** [comment]
> We should probably do that same thing we do for categories, where using the search box removes the category filter.

Do you have some example urls?  They only differences I see on a search vs. category page is we don't show the add-on type filter (because categories are already per add-on type)

**Comment by jvillalobos** [comment]
What I mean is this:

* In [a category page](https://addons.mozilla.org/firefox/extensions/category/photos-music-videos/) we have specific titles distinct from vanilla search (`Extensions in Photos, Music & Videos`).
* If you type anything in the search box, you're taken to a search page that [doesn't involve in the category anymore](https://addons.mozilla.org/firefox/search/?q=photos).
* So, while it's possible to [mix categories and search by manipulating URLs](https://addons.mozilla.org/firefox/extensions/category/photos-music-videos/?q=photos), it's not something we directly support (though for categories we do support more specific titles: `126 extensions found for "photos" in Photos, Music & Videos`).

I suggest we do the same for tags.

**Comment by eviljeff** [comment]
I'm going to aim to implement this in a similar way to category pages - with a specific url (e.g. https://addons.mozilla.org/firefox/extensions/tag/privacy/) that should be better for SEO.

**Comment by ioanarusiczki** [comment]
@eviljeff 
- Tag search results have the following format when going to them from a detail page:
 https://addons-dev.allizom.org/en-US/firefox/tag/ad blocker/ with "15 results found with tag ad blocker"

- Reading https://github.com/mozilla/addons-frontend/issues/10789#issuecomment-886907000 , similar to a search result as https://addons-dev.allizom.org/en-US/firefox/extensions/category/photos-music-videos/?q=photos I tried for tags https://addons-dev.allizom.org/en-US/firefox/tag/ad%20blocker/?q=ad%20block. As mentioned above, I added the query manually since this is not directly supported. Message for the result is:  `1 result found for "ad block"` - should it be more detailed and specify the tag too? 



- tag URLs won't work if /extensions/ is also added so perhaps the current format is expected but your comment mentioned this:
https://addons.mozilla.org/firefox/extensions/tag/privacy/

- currently there is no tag filter added to Filter results

**Comment by willdurand** [comment]
> Reading Tag filter is not visible in the search results #10789 (comment) , similar to a search result as addons-dev.allizom.org/en-US/firefox/extensions/category/photos-music-videos/?q=photos I tried for tags addons-dev.allizom.org/en-US/firefox/tag/ad%20blocker/?q=ad%20block. As mentioned above, I added the query manually since this is not directly supported. Message for the result is: 1 result found for "ad block" - should it be more detailed and specify the tag too?

ha right, I think we need a new issue specific to that. Quoting @jvillalobos above:

> So, while it's possible to mix categories and search by manipulating URLs, it's not something we directly support (though for categories we do support more specific titles: 126 extensions found for "photos" in Photos, Music & Videos).

It's only about adding some more localized strings (and tests).

**Comment by eviljeff** [comment]
>     * tag URLs won't work if /extensions/ is also added so perhaps the current format is expected but your comment mentioned this:
>       https://addons.mozilla.org/firefox/extensions/tag/privacy/

Yep, I'm aware - I started with the intention of having it under /extensions/ or /themes/ but I realised it was more like /search/, that isn't under an add-on type.

**Comment by eviljeff** [comment]
and what @willdurand said about supporting more combinations - can we have a follow-up issue please?

**Comment by AlexandraMoga** [comment]
Filed #10847 as a follow up
